src/test/java/com/wikia/webdriver/common/core/Helios.java

```diff
@@ -34,24 +36,55 @@
 public class Helios {
 
   public static HashMap<String, String> tokenCache = new HashMap<String, String>();
+  private static RequestConfig requestConfig = RequestConfig.custom().setConnectTimeout(3000)
+      .setSocketTimeout(3000).build();
 
 
-  private Helios() {
-  }
+  private Helios() {}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)